Data.TRAMS/Mappers/Request/InternalProjectToUpdateMapper.cs

```diff
@@ -136,5 +149,18 @@ private static AcademyTransferProjectBenefits Benefits(Project input)
                 EqualitiesImpactAssessmentConsidered = input.Benefits.EqualitiesImpactAssessmentConsidered
             };
         }
+        private static string ToDescription(ThreeOptions? source)
```

Comment:
It looks likes there is an extension method which is doing the same thing.

Reply:
My thoughts were that since the extension was in the web project, I didn't want a dependency from our Data project facing outwards towards web. It didn't feel right for the data project to have an extensions folder as its responsibility is data and but since we don't have a domain layer where this logic would sit I think I'll move it to the data layer anyway.

Long winded thought process - TL:DR: I'll move the existing and delete duplication.

Reply:
Have moved anyways! tech debt for a common project isn't the worst idea :)